app/src/test/java/ru/vssemikoz/newsfeed/usacases/GetFilteredNewsUseCaseTests.java

```diff
@@ -0,0 +1,93 @@
+package ru.vssemikoz.newsfeed.usacases;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+import ru.vssemikoz.newsfeed.data.NewsStorage;
+import ru.vssemikoz.newsfeed.models.Category;
+import ru.vssemikoz.newsfeed.models.Filter;
+import ru.vssemikoz.newsfeed.models.NewsFeedParams;
+import ru.vssemikoz.newsfeed.models.NewsItem;
+import ru.vssemikoz.newsfeed.usecases.GetFilteredNewsUseCase;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyBoolean;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class GetFilteredNewsUseCaseTests {
```

Comment:
A test class usually ends up with `Test`. BTW, what's happened with the code style? Don't forget to use the auto-format. This comment applies to all the test classes.